test/browser_tests/conf.js

```diff
@@ -70,7 +70,11 @@ function _retrieveProtractorParams( params ) { // eslint-disable-line complexity
   var parsedParams = {};
 
   if ( _paramIsSet( params.specs ) ) {
-    parsedParams.specs = environment.specsBasePath + params.specs;
+    var specsArray = params.specs.split( ',' );
+    for ( var i = 0, len = specsArray.length; i < len; i++ ) {
```

Comment:
Hmm, we could, but it seems like creating the function for forEach may be overkill since it's just one line in the loop?